check/setup_test.go

```diff
@@ -126,12 +127,15 @@ func Test_findPartitionID_WhenTopicDoesNotExistAndMayCreateIt_CreatesTopic(t *te
 	defer ctrl.Finish()
 
 	check, zookeeper := newZkTestCheck(ctrl)
-	mockBroker(check, ctrl)
+	connection, _, _, _ := mockBroker(check, ctrl)
 	metadata := metadataWithoutTopic()
 
 	zookeeper.EXPECT().Connect([]string{"localhost:2181"}, gomock.Any()).Return(nil, nil)
-	zookeeper.mockSuccessfulPathCreation("/config/topics/health-check")
-	zookeeper.mockSuccessfulPathCreation("/brokers/topics/health-check")
+
+	zookeeper.EXPECT().Lock(fmt.Sprintf("/healthcheck/%s", MainLockPath))
+	zookeeper.EXPECT().Unlock(fmt.Sprintf("/healthcheck/%s", MainLockPath))
+
+	connection.EXPECT().CreateTopic(gomock.Any(), gomock.Any(), gomock.Any())
```

Comment:
same thing of above here, if we can be more restrictive without hard-coding information it would be awesome :)